client/components/public/FAQ.jsx

```diff
@@ -145,8 +170,27 @@ FAQ = class FAQ extends Component {
             <br/>
 
             <iframe frameBorder="0" height="450" src="https://www.google.com/maps/embed/v1/place?key=AIzaSyDzFT6fltUNTF7Vas25IJmMkUAa5yVPi4I&amp;q=Campus+Services+Bellingham+WA" width="100%" />
+          </Accordion.Content> */}
+          <Accordion.Title active={activeIndex === 1} index={1} onClick={(e,p) => this.handleClick(e,p)} >
+            <Icon color="red" size="huge" name="dropdown"/>
+            <Icon name="info"/>
+            Safety
+          </Accordion.Title>
+          <Accordion.Content active={activeIndex === 1}>
+            In support of safety, teammates:
+            <ul>
+              <li>From different households may connect via a video conferencing platform (Zoom, Discord, Skype, &#8230;)</li>
```

Comment:
also missing a "." at the end of this line

Reply:
Fixed!